### PR TITLE
feat: Add SliverAppBar.large

### LINKED ISSUE #233: セッション詳細ページにMedium top app barを適用する
**Is your feature request related to a problem? Please describe.**
https://m3.material.io/components/top-app-bar/specs
セッションタイトルをTop App Barに表示する対応。タイトルの長さに応じて、Large styleの方が良さそうであれば、MediumではなくLargeを指定する。

### PR DESCRIPTION
**Pull request by koji-1009**
## Description

* Using SliverAppBar
* Moved the Open Twitter button to the PopMenu

Fixes #233 

## Type of change

- [x] New feature

## How Has This Been Tested?

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] My code has been formatted with `flutter format lib`.
- [ ] My code has been fixed with `dart fix --apply lib`.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.

### CODE REVIEW

File: lib/l10n/localization_en.dart [hunk 1]
```diff
@@ -129,6 +129,9 @@ final class LocalizationEn extends Localization {
   @override
   String get shareX => 'Share with X';
 
+  @override
+  String get speakerX => 'Speaker (X)';
```
Comment (Kurogoma4D) [inline lib/l10n/localization_en.dart:133]: 他の項目と文体を合わせたほうが自然かと思います。
たとえば `Open speaker profile (X)` （日本語なら `登壇者の情報を表示（X）` ）

Comment (koji-1009) [reply lib/l10n/localization_en.dart:133]: ありがとうございます！ それぐらいの長さなら、メニューに収まりそうです！

Comment (koji-1009) [reply lib/l10n/localization_en.dart]: [e986bd3](https://github.com/FlutterKaigi/conference-app-2023/pull/259/commits/e986bd3da41b5d386a96afed7cfc375b2a361842)にて修正しました 🙏 感謝です。

**Review by Kurogoma4D — APPROVED** [review]
👍

**Pull request merged**